examples/nci/nci_rf.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script that trains Sklearn multitask models on nci dataset.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
+import os
+import numpy as np
+import shutil
+from sklearn.linear_model import LogisticRegression
+from sklearn.ensemble import RandomForestClassifier
+from deepchem.datasets import Dataset
+from deepchem.models.multitask import SingletaskToMultitask
+from deepchem import metrics
+from deepchem.metrics import Metric
+from deepchem.models.sklearn_models import SklearnModel
+from deepchem.utils.evaluate import Evaluator
+from deepchem.datasets.nci_datasets import load_nci
+
+np.random.seed(123)
+
+# Set some global variables up top
+
+reload = False
+verbosity = "high"
+
+#base_data_dir = "/scratch/users/rbharath/muv"
```

Comment:
Can you remove the commented out `base_data_dir`? In general, we need to figure out a better way to handle base directories in these examples. Maybe just create the base directory in the same directory as the script? (No need to handle in this PR though, just some general thoughts)